src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,164 @@
+.header {
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 260px;
+  }
+
+  &__add {
+    background: #52106a;
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 0 15px;
+
+    h4 {
+      font-family: "Avenir Next";
+      font-style: normal;
+      font-weight: 500;
+      font-size: 11px;
+      line-height: 18px;
+
+      letter-spacing: 2.2px;
+      text-transform: uppercase;
+
+      color: #ffffff;
+    }
+  }
+
+  &__main {
+    height: 120px;
+    min-width: 100%;
+    display: grid;
+    grid-template-columns: 100px 264px 100px;
+    justify-content: space-between;
+    align-items: center;
+    grid-template-columns: repeat(3, 120px);
+
+    div:first-child {
+      justify-self: start;
+    }
+
+    img {
+      justify-self: center;
+      height: 33px;
+      width: 264px;
+    }
+
+    div:last-child {
+      justify-self: end;
+    }
+  }
+
+  &__links {
+    overflow: scroll;
+    max-width: 100%;
+    ul {
+      display: flex;
+      li:not(:first-child) {
+        margin-left: 40px;
+      }
+    }
+  }
+
+  &__buttons {
+    display: flex;
+    align-items: center;
+    button:not(:first-child) {
+      margin-left: 30px;
+    }
+  }
+
+  hr {
+    width: 100%;
+    height: 2px;
+    background: #d4d4d4;
+  }
+}
+
+@media screen and (max-width: 1440px) {
+  .header {
+    &__content {
+      margin: 0 60px;
+    }
+  }
+}
+
+@media screen and (max-width: 980px) {
+  .header {
+    &__content {
+      margin: 0 30px;
+      height: 140px;
+    }
+
+    &__languages {
+      display: none;
+    }
+
+    &__links {
+      display: none;
+    }
+
+    &__main {
+      height: 140px;
+      grid-template-columns: repeat(3, 140px);
+    }
+  }
+}
+
+@media screen and (max-width: 620px) {
+  .header {
+    &__buttons {
+      button:nth-child(2) {
+        display: none;
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 520px) {
+  .header {
+    &__content {
+      margin: 0 15px;
+      height: 104px;
+    }
+    &__add {
+      height: 44px;
+
+      h4 {
```

Comment:
do not add style to an empty tag

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/e1b8f39f5217f487124296a507c3e31c860d8306